keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/image/ArtifactVersionDeployed.kt

```diff
@@ -17,12 +17,14 @@
  */
 package com.netflix.spinnaker.keel.ec2.image
 
+import com.netflix.spinnaker.keel.api.ResourceId
+
 /**
  * An event fired while we check a server group that lets us know
  * what version of software is running in that server group
  */
-data class ArtifactAlreadyDeployedEvent(
-  val resourceId: String,
+data class ArtifactVersionDeployed(
```

Comment:
I want to move to this event being the sole trigger for a version being considered successfully deployed so renaming it seemed the right thing.